servers/services/src/test/java/org/projectnessie/services/rest/TestJerseyRest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.rest;
+
+import javax.ws.rs.core.Application;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.jboss.weld.environment.se.Weld;
+import org.jboss.weld.environment.se.WeldContainer;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.projectnessie.services.config.ServerConfigExtension;
+import org.projectnessie.versioned.VersionStoreExtension;
+
+public class TestJerseyRest extends AbstractTestRest {
+  private Weld weld;
+  private JerseyTest jerseyTest;
+
+  @Override
+  @BeforeEach
```

Comment:
I think, this is different behavior from the original `TestRest` / `ITNativeRest`, which starts one Nessie/Quarkus for the whole test class - aka `@BeforeAll`. Can change this to `@BeforeAll`/`@AfterAll`?

Reply:
sure I can change that. I don't know why Laurent chose before each/after each